src/engraving/libmscore/slur.cpp

```diff
@@ -1327,7 +1327,7 @@ SpannerSegment* Slur::layoutSystem(System* system)
                 // slurs go above if there are mixed direction stems between c1 and c2
                 // but grace notes are exceptions
                 _up = true;
-            } else if (m1->hasVoices(startCR()->staffIdx(), tick(), ticks()) && c1 && !c1->isGrace()) {
+            } else if (m1->hasVoices(startCR()->staffIdx(), tick(), ticks() + c2->ticks()) && c1 && !c1->isGrace()) {
```

Comment:
Are you sure that we don't need a nullptr check for `c2` here? Maybe wrap the whole if/elseif block in an `if (c1 && c2)` block?

Reply:
oh, good call!

Reply:
We still need to get the tick / ticks for that CR, so I'll change it to check the ChordRest instead of the Chord